jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -49,9 +49,6 @@
     static final String INSERT_NO_FIELD = "INSERT INTO Test VALUES (1, 25, 'Santa', 'Claus', 'jibberish')";
     static final String INSERT_WRONG_TYPE = "INSERT INTO Test VALUES ('string', 'string', 3, 4)";
     
-    DateFormat dfTimestamp  = new SimpleDateFormat("yyyy-dd-mm'T'HH:mm:ss.SSSZ");
-    DateFormat dfDate       = new SimpleDateFormat("yyyy-dd-mm");
-    DateFormat dfTime       = new SimpleDateFormat("HH:mm:ss.SSSZ");
     static final String timestampPattern = "\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}.\\d{3}-\\d{4}";
     static final String datePattern = "\\d{4}-\\d{2}-\\d{2}";
     static final String timePattern = "\\d{2}:\\d{2}:\\d{2}.\\d{3}-\\d{4}";
```

Comment:
I think we need to extend the test here.  This test passed before but wasn't correct.  We should track what date we put in & verify it that we get the same thing (or the expected thing) back.  As the test is, I don't think it will verify that the date is correct does it?  I think we just ask if it _looks like a date_.

Reply:
Yes, it's best to run your test WITHOUT the fix, to make sure the test fails and it catches the problem. If it doesn't fail, then you need to improve the test.

THEN run the test with the fix to make sure it passes with the fix.